i_elks.c

```diff
@@ -79,15 +79,14 @@ static void I_SetScreenMode(uint16_t mode)
   "push %%bp;"
   "push %%es;"
   "cli;"
-  "mov %%ax,%0;"
   "int $0x10;"
   "sti;"
   "pop %%es;"
   "pop %%bp;"
   "pop %%di;"
   "pop %%si;"
      : /* no outputs */
-     : "r" (mode)
+     : "a" (mode)
```

Comment:
So this line will load the mode variable into the ax register?

Reply:
yes